impress-org/givewp
# Refactor: addons admin page to use react-aria-components instead of @reach/tabs

## Description



[Reach UI is not currently maintained](https://github.com/reach/reach-ui/issues/972) and will not be supporting later version of React.  This PR uses a recommended alternative library called `react-aria-components` as a _drop in_ replacement for our usage of @reach/tabs in the addon admin page.

## Affects



The addon admins page

## Visuals



<img width="1436" alt="Screenshot 2023-08-16 at 9 43 07 AM" src="https://github.com/impress-org/givewp/assets/10138447/640f0b19-4f02-42ef-a114-f1016fb7bbc7">


## Testing Instructions



- Build assets (might need to use `--legacy-peer-deps` for now
- View the "Add-ons" page
- Make sure it looks the same

## Pre-review Checklist
